uber/templates/regform.html

```diff
@@ -256,9 +256,6 @@
                             width: '100%',
                         }).val({{ attendee.affiliate|default('')|jsonize }}).trigger('change');
                     }
-                    $('form').submit(function(){
-                        $(':submit').attr('value', 'Uploading Preregistration...').attr('disabled', true);
-                    });
```

Comment:
This was actually in the wrong place anyway. It was inside the `extra_donation` if statement. Furthermore, `.attr('value', 'Uploading Preregistration...')` doesn't actually change the text of the button (which I presume was the point).